app/multitenant/aws_collector.go

```diff
@@ -537,63 +659,40 @@ func (c *awsCollector) Add(ctx context.Context, rep report.Report, buf []byte) e
 		return err
 	}
 
-	// first, put the report on s3
-	rowKey, colKey := calculateDynamoKeys(userid, time.Now())
-	reportKey, err := calculateReportKey(rowKey, colKey)
-	if err != nil {
-		return err
-	}
-
 	// Shortcut reports are published to nats but not persisted -
 	// we'll get a full report from the same probe in a few seconds
-	if !rep.Shortcut {
-		reportSize, err := c.cfg.S3Store.StoreReportBytes(ctx, reportKey, buf)
-		if err != nil {
-			return err
-		}
-		reportSizeHistogram.Observe(float64(reportSize))
-		reportSizePerUser.WithLabelValues(userid).Add(float64(reportSize))
-		reportsPerUser.WithLabelValues(userid).Inc()
-	}
-
-	// third, put it in memcache
-	if c.cfg.MemcacheClient != nil {
-		_, err = c.cfg.MemcacheClient.StoreReportBytes(ctx, reportKey, buf)
-		if err != nil {
-			// NOTE: We don't abort here because failing to store in memcache
-			// doesn't actually break anything else -- it's just an
-			// optimization.
```

Comment:
Why not keep this comment?

Reply:
Ah I see, now we actually do abort here - any reason why that `return nil` was added?

Reply:
I have added it back, also another comment on the case where a shortcut report can't be stored.